public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.test.ts

```diff
@@ -1,10 +1,15 @@
+import { Monaco, monacoTypes } from '@grafana/ui/src';
+
+import MonacoMock from '../../../../cloudwatch/__mocks__/monarch/Monaco';
 import LokiLanguageProvider from '../../../LanguageProvider';
 import { LokiDatasource } from '../../../datasource';
 import { createLokiDatasource } from '../../../mocks';
 
 import { CompletionDataProvider } from './CompletionDataProvider';
 import { getAfterSelectorCompletions, getCompletions } from './completions';
-import { Label, Situation } from './situation';
+import { getSituation, Label, Situation } from './situation';
+
+import { calculateRange } from './index';
```

Comment:
It's a nit, but I'm not really sure this is the file for these tests. `index.ts` is kinda awkward, so your choice. Maybe it would make sense to rename this `index.ts` to something else, as it's not a super popular pattern in the repo AFAIK. A follow-up PR would be fine as well.

Reply:
I created a chore to deal with this https://github.com/grafana/grafana/issues/76374